android/app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -24,66 +26,48 @@
     private String name;
     private Double latitude;
     private Double longitude;
-    @JsonProperty("location-name")
     private String locationName;
-    @JsonProperty("starts-at")
-    private String startTime;
-    @JsonProperty("ends-at")
-    private String endTime;
+    private String startsAt;
```

Comment:
I think `deleteRealmIfMigrationNeeded()` will be best solution. Using it Realm will be cleared and recreated with the new Realm schema. And also there is no method to temporarily delete db if migration needed